src/Interface/Lists/Disforge.ts

```diff
@@ -0,0 +1,66 @@
+import { Service, ServicePostOptions } from '../Service'
+import { Util } from '../../Utils/Util'
+
+/**
+ * Represents the Disforge service.
+ * @see https://disforge.com/developer
+ */
+export default class Disforge extends Service {
+  /** The values that can be used to select the service. */
+  static get aliases() {
+    return ['disforge', 'df', 'disforge.com']
+  }
+
+  /** The logo URL. */
+  static get logoURL() {
+    return 'https://disforge.com/assets/img/ui/categories/all-bots.png'
+  }
+
+  /** Service's name. */
+  static get serviceName() {
+    return 'Disforge'
+  }
+
+  /** The website URL. */
+  static get websiteURL() {
+    return 'https://disforge.com/bots'
+  }
+
+  /** The base URL of the service's API. */
+  static get baseURL() {
+    return 'https://disforge.com/api'
+  }
+
+  /**
+   * Posts statistics to this service.
+   * <warn>Shard data posting is not supported for this service.</warn>
+   * @param options The options of the request
+   */
+  static post(options: ServicePostOptions) {
+    const { token, clientID, serverCount } = options
+    return super._post({
+      method: 'post',
+      url: `/botstats/${Util.resolveID(clientID)}`,
+      headers: {
+        Authorization: token
+      },
+      data: {
+        servers: Util.resolveCount(serverCount)
```

Comment:
The docs don't say whether `servers` should be a payload property, a query parameter, or a header... I'm just going to assume it's the first, since my bot hasn't been approved yet.